deployment/create_deployment_trigger.sh

```diff
@@ -0,0 +1,23 @@
+# !/bin/bash
+
+source args.sh
+
+PUB_SUB_TOPIC="deploy-index-server"
+gcloud pubsub topics create --project=$PROJECT_ID $PUB_SUB_TOPIC
+
+# Create a periodic task that will trigger a new deployment 9AM UTC every
+# wednesday.
+gcloud scheduler jobs create pubsub --project=$PROJECT_ID \
+  "deployment-scheduler" --schedule="0 9 * * 3" --topic=$PUB_SUB_TOPIC \
+  --message-body="Deploy"
+
+set +x
+# TODO: Create this automatically through gcloud cli once it is possible.
+echo "You need to create a build trigger that'll listen on $PUB_SUB_TOPIC and"
```

Comment:
Since this is quite important, maybe shout this to the user somehow?

```sh
RED='\033[0;31m'
NC='\033[0m' # No Color
echo '{RED}WARNING{NC}'
```

warning/importotant/note/whatever else might help here.